app/controllers/cinema_halls_controller.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class CinemaHallsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    cinema_halls = CinemaHall.all
+
+    render json: cinema_halls
+  end
+
+  def show
+    CinemaHall.find(params[:id])
```

Comment:
You forgot to assign it to the variable, so your render with return nothing now.

Reply:
Thanks, corrected 👍